### PR TITLE
Show Python version in EXE metadata

### PR DESCRIPTION
**Pull request by nihil-admirari**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Currently there is no way of knowing what Python version provided `exe`s are bundled with. In particular, it's hard to find out whether the latest security fixes are included. To solve it, this pull request adds Python version to product version string.

### CODE REVIEW

**Comment by pukkandan** [comment]
If you have small PRs like this and #383, you can just open a single PR with all of them if you want and I can cherrypick the commits

File: pyinst.py [hunk 1]
```diff
@@ -58,7 +59,7 @@
                     ),
                     StringStruct('OriginalFilename', 'yt-dlp%s.exe' % _x86),
                     StringStruct('ProductName', 'yt-dlp%s' % _x86),
-                    StringStruct('ProductVersion', '%s%s' % (VERSION, _x86)),
+                    StringStruct('ProductVersion', '%s%s on Python %s' % (VERSION, _x86, PY_VER)),
```
Comment (pukkandan) [inline pyinst.py:62]: Why not just

```suggestion
                    StringStruct('ProductVersion', '%s%s on Python %s' % (VERSION, _x86, platform.python_version())),
```

**Comment by pukkandan** [comment]
also, on a sidenote, you can tell the exact python version by simply running `yt-dlp -v`

File: pyinst.py (hunk 1 above)
Comment (nihil-admirari) [reply pyinst.py]: Didn't know about it. Fixed.

File: pyinst.py [hunk 2]
```diff
@@ -58,7 +58,9 @@
                     ),
                     StringStruct('OriginalFilename', 'yt-dlp%s.exe' % _x86),
                     StringStruct('ProductName', 'yt-dlp%s' % _x86),
-                    StringStruct('ProductVersion', '%s%s' % (VERSION, _x86)),
+                    StringStruct(
+                        'ProductVersion',
+                        '%s%s on Python %s' % (VERSION, _x86, (platform.python_version()))),
```
Comment (pukkandan) [inline pyinst.py:63]:
```suggestion
                        '%s%s on Python %s' % (VERSION, _x86, platform.python_version())),
```

**Comment by nihil-admirari** [comment]
> If you have small PRs like this and #383, you can just open a single PR with all of them if you want and I can cherrypick the commits

Ok, will do in the future. Let's just leave these two for now.

> also, on a sidenote, you can tell the exact python version by simply running `yt-dlp -v`

True, but not obvious.

Btw, why `yt-dlp.exe` uses 3.8.10 instead of 3.9.5?

**Comment by pukkandan** [comment]
> Btw, why yt-dlp.exe uses 3.8.10 instead of 3.9.5?

Windows 7 compatibility

**Pull request merged**